cms/migrations/0049_benefitspage_content_table_block.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.2.13 on 2020-10-26 09:12
+
+from django.db import migrations
+import wagtail.contrib.table_block.blocks
+import wagtail.core.blocks
+import wagtail.core.fields
+import wagtail.images.blocks
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cms', '0048_convert_description_richtextfield_to_streamfield'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='benefitspage',
+            name='content',
+            field=wagtail.core.fields.StreamField([('rich_text', wagtail.core.blocks.RichTextBlock()), ('image_with_link', wagtail.core.blocks.StructBlock([('image', wagtail.images.blocks.ImageChooserBlock(help_text='The image to display.', label='Image', required=True)), ('link', wagtail.core.blocks.URLBlock(help_text='Absolute URL to the image, like https://example.com/some_image.jpg', label='Link', required=True)), ('align', wagtail.core.blocks.ChoiceBlock(choices=[('center', 'Center'), ('right', 'Right'), ('left', 'Left')], max_length=10)), ('width', wagtail.core.blocks.IntegerBlock(required=False)), ('height', wagtail.core.blocks.IntegerBlock(required=False))], blank=True, help_text='Upload an image with a clickable link')), ('benefits_table', wagtail.contrib.table_block.blocks.TableBlock(table_options={'renderer': 'html', 'startCols': 2, 'startRows': 3}))], blank=True, help_text='The content of the benefits page'),
```

Comment:
I'm not too familiar with the Wagtail schema, but there seem to be a lot more "blocks" here than I was expecting. 
Are they all necessary? Also, this line is super-long. Does it pass the linter?

@annagav do we need a reverse migration? 

Also, please check with @umarmughal824 that this will be compatible with his Wagtail 2.9.3 upgrade.

Reply:
@pdpinch Since we changed the `content` field in the benefits page model and that being the `StreamField` containing multiple sub-blocks e.g. RichText, Image with link & (Now benefits table) so this auto-generated migration was expected to have these blocks change. 
Additionally, this is being passed by the linter, but I'll try to format it though.
On the other point, @annagav can provide insights but since the table block is newly added and I suppose we might not need reverse migrations.
Let me know your thoughts.

Reply:
I'm ok with it if @annagav is OK with it.

Reply:
This doesn't need a reverse migration.